roles/lnls-ans-role-sirius-hlacons/README.md

```diff
@@ -0,0 +1,86 @@
+LNLS Ansible Role Sirius HLA Apps
+=================================
+
+This Ansible role configures some defaults Sirius High Level Applications for Sirius Light Source control machines.
+
+## Requirements
+
+- ansible >= 2.4
+- molecule >= 2.20
+
+## Role Variables
+
+
+## Example Playbook
+
+```yaml
+---
+- hosts: all
+  tasks:
+  - import_role:
+      name: '{{playbook_dir}}'
+```
+
+## Example Commmand
+
+```bash
+ansible-playbook -i host, -u user -k --ask-become-pass playbook.yml
+```
+
+## Tests
+
+Tests are performed using Molecule. To run them with python virtualenv, issue:
+
+```bash
+    bash -c "\
+        cd ../ && \
+        virtualenv env --python python3 && \
+        source env/bin/activate && \
+        cd lnls-ans-role-sirius-hla && \
+        pip install molecule docker-py && \
```

Comment:
molecule, as of version 3.0, does not install test frameworks as a dependency. We must install them separately with:
pip install molecule testinfra yamllint ansible-lint flake8 docker-py